administrator/language/en-GB/com_installer.ini

```
@@ -25,8 +25,12 @@ COM_INSTALLER_DISCOVER_TABLE_CAPTION="Table of Discovered Extensions"
 COM_INSTALLER_DOWNLOADKEY_EXTRA_QUERY_LABEL="Download Key"
 COM_INSTALLER_DOWNLOADKEY_MISSING_LABEL="The Download Key is missing"
 COM_INSTALLER_DOWNLOADKEY_MISSING_TIP="Updates for this extension will not work until you enter its Download Key. Click on the Update Site's title to enter the Download Key."
+COM_INSTALLER_EMPTYSTATE_BUTTON_ADD="Discover extensions to install"
+COM_INSTALLER_EMPTYSTATE_CONTENT="We have not found any extension on your site, that have not gone through the normal installation process."
```

```suggestion
COM_INSTALLER_EMPTYSTATE_CONTENT="We have not found any extensions on your site that have not gone through the normal installation process."
```

pluralise and remove comma

Not sure about the "We .." as that is not consistent with any other message on the site. Who is we?

---

[Like I said](https://github.com/joomla/joomla-cms/pull/33400#issuecomment-828831156) "And yes I hate the title and sentences too! - please suggestion new ones if you dont like them too. I'll not be offended."